include/revng/Storage/Path.h

```diff
@@ -0,0 +1,93 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "llvm/ADT/SmallString.h"
+#include "llvm/Support/Path.h"
+
+#include "revng/Storage/StorageClient.h"
+#include "revng/Support/Debug.h"
+
+namespace revng {
+
+class Path {
+private:
+  revng::StorageClient *Client;
```

Comment:
As a policy, always initialize primitive types and pointers inline. Add `= nullptr`.

Reply:
Added and double-checked everywhere else